src/css/components/grid.scss

```diff
@@ -1,7 +1,12 @@
 .grid {
-  @include outer-container;
```

Comment:
I think the correct thing here is to change the values [here](https://github.com/18F/cg-style/blob/master/src/css/vars.scss#L9) to the correct value. I think you might have to check that it doesn't break cg-site.